apps/open-justice/aleph.yml

```diff
@@ -0,0 +1,17 @@
+---
+hosts: all
```

Comment:
use the app group name instead of all - we don't want to clone aleph on all hosts - just the app host in each environment (currently prod)